### PR TITLE
minor doc fixes

### PR DESCRIPTION
**Pull request by erictu**

### CODE REVIEW

File: docs/installation/distribution.rst [hunk 1]
```diff
@@ -57,3 +57,9 @@ And bdgenomics.mango:
 .. code:: bash
 
     pip install bdgenomics.mango
+
+And bdgenomics.adam:
```
Comment (akmorrow13) [inline docs/installation/distribution.rst]: This shouldn't be necessary, bdgenomics.adam is a dependency of bdgenomics.mango

Comment (erictu) [reply docs/installation/distribution.rst]: Sounds good, double checked the pip install and removed these lines.

**Comment by akmorrow13** [comment]
Jenkins, retest this please.

**Comment by akmorrow13** [comment]
Thanks @erictu!

**Pull request merged**